cornice_swagger/swagger.py

```diff
@@ -276,14 +283,18 @@ class DefinitionHandler(object):
 
     json_pointer = '#/definitions/'
 
-    def __init__(self, ref=0):
+    def __init__(self, ref=0, typ_dispatcher=TypeConversionDispatcher()):
```

Comment:
Is there a good reason to have to overwrite the whole `TypeConversionDispatcher` class? Can't we just pass a dict mapping the user defined cornice types to a callable implementing the `cornice_swagger.converters.schema.TypeConverter` interface?